src/containers/navigation-icons/NavigationIcons.styles.ts

```diff
@@ -14,5 +15,14 @@ export const styles = {
     ...hideOnMobile,
     ml: '12px'
   },
-  studentIcons: { ...hideOnMobile, color: 'primary.900' }
+  studentIcons: {
+    ...hideOnMobile,
+    color: 'primary.900'
+  },
+  accountIcon: {
+    ...hideOnMobile,
+    color: 'primary.900',
+    ml: { sm: '12px', md: '7px', lg: '12px' },
```

Comment:
Why you can not use 12px for `md` too and what it will have for `xs`

Reply:
It can be! Changed. 
It dosnt display on sm-screen, because of that, I think,  we don't need ml for xs